packages/domain/src/use-cases/profile/__tests__/ActiveProfileLoader.spec.ts

```diff
@@ -57,7 +57,7 @@ describe(`Given the ${ActiveProfileLoader.name} interactor`, () => {
       expect(activeProfilePresenter.presentActiveProfile).toHaveBeenCalledWith(profile);
     });
 
-    it('should bail out if not active profile is present and the wallet does not own any profile', async () => {
+    it('should throw if not active profile is present and the wallet does not own any profile', async () => {
```

Comment:
I think we need to account for login with wallet that do not have an profile (cause they cannot claim one at first or because they have to go through onboarding once open profile minting will be enable).

So throwing is probably not necessary.